src/Stats/Provider/StatsApiProvider.php

```diff
@@ -43,6 +43,7 @@ public function __construct(array $config, callable $handler = null)
         }
 
         $this->apiKey = $config['statsApi']['apiKey'];
+        $this->userAgent = $config['statsApi']['userAgent'];
```

Comment:
can we specify a default one? otherwise we need to ad the param in ansible, MonarcAppFO (config, vargant, update docs) and packer!

Reply:
Already changed [here](https://github.com/monarc-project/MonarcAppFO/pull/342/files) for some parts.
But I can make it work in a different way if your prefer.

Reply:
I think yes, we need to bypass the real user agent, and if it's a cronjob / cli script then something generic.